### ISSUE TITLE
SVG imports not working with yarn rw test

### ISSUE THREAD

**Issue by RobertBroersma** [redwoodjs/redwood#391]
Jest doesn't know how to handle .svg imports.

A mock transformer should be implemented.

What I usually do is just transform the SVG into an empty module (in this case it needs to be something react-y I assume since `svg-react-loader` transforms SVG's into Components.

What is a good generic way to transform SVG's like that? Is stubbing them out to be nothing too limiting for some people who want to assert on the SVG?

Happy to create a PR if we can clear this up!

**Comment by thedavidprice** [comment]
Curious if you're implementing Jest tests currently or simply thinking ahead? Asking because we've yet to finish implementing built-in Jest support -- yes, you can run `yarn rw test` but as soon as you start importing it's failing all over the place. So if you're already up and running on your own, let's talk! 😀

Back to your question, yes, I think the way forward here will be `jest.mock()`. I'll fully defer to @peterp however.

**Update on CRA Jest support:**
Peter and Rob are just about done with [test implementation for our CLI package](https://github.com/redwoodjs/redwood/pull/401). We'll use the fixtures pattern for CRA implementation as a helper/utility. For examples see [here](https://github.com/redwoodjs/redwood/blob/pp-add-tests-for-cli-generators/packages/cli/src/lib/test.js) and [here](https://github.com/redwoodjs/redwood/tree/pp-add-tests-for-cli-generators/packages/cli/src/lib/__tests__). Mentioning this in the case it seems we should sync this request with next steps on CRA Jest support.

**Comment by RobertBroersma** [comment]
Ah, great!

I didn't realize it's not done yet. I figured I can run it = I'm supposed to be able to run it.

I tried running `yarn rw test` in my web package and this was the first issue that came up, but I'll certainly wait for the PR you linked to be merged!

PS. I think using the [moduleNameMapper](https://jestjs.io/docs/en/configuration#modulenamemapper-objectstring-string--arraystring) or maybe [transformer](https://jestjs.io/docs/en/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object) option in Jest Config would be simpler than using `jest.mock()`
